distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Split_Tokenize.enso

```diff
@@ -0,0 +1,246 @@
+from Standard.Base import all
+
+import project.Data.Column.Column
+import project.Data.Table.Table
+import project.Internal.Java_Exports
+import project.Internal.Problem_Builder.Problem_Builder
+import project.Internal.Unique_Name_Strategy.Unique_Name_Strategy
+
+from project.Data.Type.Value_Type import Auto
+from Standard.Table import Value_Type
+from Standard.Table.Errors import Column_Count_Exceeded, Duplicate_Output_Column_Names, Invalid_Value_Type, Missing_Input_Columns
+from project.Internal.Java_Exports import make_string_builder
+
+polyglot java import org.enso.table.data.mask.OrderMask
+
+## PRIVATE
+   Splits a column of text into a set of new columns.
+   See `Table.split_to_columns`.
+split_to_columns : Table -> Text | Integer -> Text -> Auto | Integer -> Problem_Behavior -> Table
+split_to_columns table input_column_name delimiter="," column_count=Auto on_problems=Report_Error =
```

Comment:
```suggestion
split_to_columns table column_id delimiter="," column_count=Auto on_problems=Report_Error =
```

It may be an index as well as a name, I'd avoid a variable name that can be misleading.